src/contexts/ReviewContext.js

```diff
@@ -0,0 +1,27 @@
+import React, {useContext, useState} from 'react';
+
+const ReviewContext = React.createContext();
+
+export function useReview() {
+  return useContext(ReviewContext);
+}
+
+export function ReviewProvider({children}) {
+  const [sort, setSort] = useState('relevant');
+  const [quantity, setQuantity] = useState(2);
+
+
+  function setNewSort(sortOn) {
+    setSort(sortOn);
+  }
+
+  function setNewQuantity(num) {
+    setQuantity(num);
+  }
+
+  return (
+    <ReviewContext.Provider value = { {sort: sort, setNewSort:setNewSort, quantity: quantity, setNewQuantity: setNewQuantity }}>
```

Comment:
I learned that you can set a key in your value object to be an array as so: 
`sort: [sort, setNewSort]`
And then you can just destructure each separate value on your functional component and use them as needed.
Just a little tip! This is fine too